src/components/pages/FlagBuilder/index.js

```diff
@@ -19,21 +19,24 @@ const ContentContainer = styled.div`
 `;
 
 const SliderContainer = styled.div`
-  padding: 2rem 4rem;
+  padding: 5%;
```

Comment:
This seems to space out the page a bit too much, so that you can't really see the whole flag when using the slider. Maybe add `padding-top: 0;` as well?